code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -450,34 +450,33 @@ nobiliumsuppression = INFINITY
 			air.temperature = max(((temperature * old_heat_capacity + energy_produced) / new_heat_capacity), TCMB) //the air heats up when reacting
 		return REACTING
 
-/datum/gas_reaction/nitrylformation //The formation of nitryl. Endothermic. Requires bz.
+/datum/gas_reaction/nitryum_formation //The formation of nitryum. Endothermic. Requires bz.
 	priority = 3
-	name = "Nitryl formation"
-	id = "nitrylformation"
+	name = "Nitryum formation"
+	id = "nitryum_formation"
 
-/datum/gas_reaction/nitrylformation/init_reqs()
+/datum/gas_reaction/nitryum_formation/init_reqs()
 	min_requirements = list(
-		/datum/gas/oxygen = 10,
+		/datum/gas/tritium = 20,
 		/datum/gas/nitrogen = 10,
-		/datum/gas/bz = 5,
-		"TEMP" = 1500,
-		"MAX_TEMP" = 10000
+		/datum/gas/bz = 10,
+		"TEMP" = FIRE_MINIMUM_TEMPERATURE_TO_EXIST + 100
 	)
 
-/datum/gas_reaction/nitrylformation/react(datum/gas_mixture/air)
+/datum/gas_reaction/nitryum_formation/react(datum/gas_mixture/air)
 	var/list/cached_gases = air.gases
 	var/temperature = air.temperature
 
 	var/old_heat_capacity = air.heat_capacity()
 	var/heat_efficency = min(temperature / (FIRE_MINIMUM_TEMPERATURE_TO_EXIST * 8), cached_gases[/datum/gas/oxygen][MOLES], cached_gases[/datum/gas/nitrogen][MOLES])
-	var/energy_used = heat_efficency * NITRYL_FORMATION_ENERGY
-	ASSERT_GAS(/datum/gas/nitryl, air)
+	var/energy_used = heat_efficency * NITRYUM_FORMATION_ENERGY
+	ASSERT_GAS(/datum/gas/nitryum, air)
 	if ((cached_gases[/datum/gas/oxygen][MOLES] - heat_efficency < 0 ) || (cached_gases[/datum/gas/nitrogen][MOLES] - heat_efficency < 0) || (cached_gases[/datum/gas/bz][MOLES] - heat_efficency * 0.05 < 0)) //Shouldn't produce gas from nothing.
 		return NO_REACTION
 	cached_gases[/datum/gas/oxygen][MOLES] -= heat_efficency
```

Comment:
Ditto. Unless you want the tritium to be a catalyst, in which case you might want to readd the oxygen requirement to the initial requirements.

Reply:
Same i forgot to change oxygen to Trit, will do it in the next commit